app/assets/stylesheets/custom.scss

```diff
@@ -0,0 +1,145 @@
+.the-index-page {
+  background-image: image-url('sky-temple.jpg');
+  background-image: no-repeat;
+  background-attachment: fixed;
+  background-size: cover;
+  background-color: black;
+  position: relative;
+}
+
+//defining the unique font used across the site
+ @font-face {
+   font-family: "MKXfont";
+   src: font-url('MKXfont.ttf');
+ }
+
+//defining background images for specific character show pages. Yeah, all 25
+ @import 'character_show_pages';
+
+//the default background image
+ .the-show-page {
+   background-image: no-repeat;
+   background-attachment: fixed;
+   background-size: cover;
+   background-color: black;
+   position: relative;
+ }
+
+//top-left logo
+ .top-bar .name h4 a:before {
+   background-image: url('MKX-Logo-Icon.jpg');
+   background-repeat: no-repeat;
+   content: "[]";
+   display: inline-block;
+   height: 57px;
+   position: relative;
+   width: 60px;
+ }
+
+//panel elements
+
+ .fullWidth {
+    width: 100%;
+    margin-left: auto;
+    margin-right: auto;
+    max-width: initial;
+ }
+
+ .move-properties {
+   table {
+     margin: 1rem auto;
+   }
+ }
+
+ .column-show {
+   margin-left: 40px;
+ }
+
+ .field-log-in {
+   color: #ead61c;
+ }
+
+ .text-center {
+   text-align: center;
+ }
+
+ .panel-show {
+   background:rgba(255,255,255,0.15);
+   margin-left: 8px;
+   margin-right: 8px;
+   text-align: center;
+   border-style: outset;
+   border-width: 3px;
+   margin-bottom: 12px;
+ }
+
+ .panel-reg {
+   background-color: rgba(0,0,0,0.25);
+   margin-top: 20px;
+   margin-left: 8px;
+   margin-right: 8px;
+   }
+
+ .panelzz {
+   background-color: rgba(0,0,0,0.5);
+ }
+
+ .panelz {
+   background-color: rgba(0,0,0,0.3);
+   padding-top: 20px;
+ }
+
+ .paneled {
+   background:rgba(255,255,255,0.30);
+ }
+
+ .panel {
+   background:rgba(255,255,255,0.30);
+   margin-left: 8px;
+   margin-right: 8px;
+ }
+
+ .index_row {
+   max-width: 90rem;
+ }
+
+ .index-name {
+   text-align: center;
+ }
+
+ .index-pic {
+   display: block;
+   margin-left: auto;
+   margin-right: auto;
+ }
+
+ #register-button {
+   column-rule-color: rgba(0,0,0,0.30);
+   background-color: rgba(0,0,0,0.30);
+   margin-right: 1px;
+ }
+
+ #register-button-2 {
+   background-color: rgba(0,0,0,0.30);
+   margin-right: 1px;
+ }
+
+//button effects
+ .hvr-bounce-in {
+   display: inline-block;
```

Comment:
Properties should be ordered -webkit-backface-visibility, backface-visibility, box-shadow, display, -moz-osx-font-smoothing, -webkit-transform, transform, -webkit-transition-duration, transition-duration, vertical-align